sdk/canton/community/ledger/ledger-json-api/src/main/scala/com/digitalasset/canton/http/json/ResponseFormats.scala

```diff
@@ -42,7 +42,10 @@ object ResponseFormats {
       jsVals ~> partition.in
 
       // second consume all successes
-      partition.out1.zipWithIndex.map(a => formatOneElement(a._1, a._2)) ~> concat.in(1)
+      partition.out1
+        .zip(Source.fromIterator(() => Iterator.iterate(0L)(_ + 1)))
+        .map(a => formatOneElement(a._1, a._2)) ~> concat.in(1)
```

Comment:
Are these not the same?

Reply:
https://github.com/apache/pekko/issues/1525

Reply:
Ah, perhaps link this issue so someone can fix it at a later date if they're making changes here